pkg/apis/core/v1/defaults.go

```diff
@@ -69,6 +69,9 @@ func SetDefaults_Volume(obj *v1.Volume) {
 			EmptyDir: &v1.EmptyDirVolumeSource{},
 		}
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.ImageVolume) && obj.Image != nil && obj.Image.PullPolicy == "" {
```

Comment:
We need the "latest" defaulting behavior if image pull policy is empty, and test case.

Reply:
Thanks, fixed as suggested :+1: